### PR TITLE
custom options for inquirer prompt

### PR DESCRIPTION
**Pull request by chamathb**
Support for inquirer js prompt's custom configurations

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,17 +30,24 @@ const getState = (key) => {
...
+  const answer = await inquirer.prompt([{ ...promptObj, prefix: meta.prefix, message }]);
+  return answer.value;
+};
+
 // reusable prompt; wraps inquirer
 // e.g. const name = await prompot('What is your name?');
-const prompt = async (message) => {
-  const answer = await inquirer.prompt([{ ...meta.prompt, prefix: meta.prefix, message }]);
-  return answer.value;
+const prompt = async (message, customConf = {}) => {
```
Comment (hliyan) [inline index.js]: can you rename `customConf` to `options`? that's the usual name.

File: test.js [hunk 2]
```diff
@@ -7,6 +7,10 @@ const login = async (args) => {
   const password = await cli.promptPassword('password:');
 };
 
+const promptWithDefaults = async (args) => {
+  const lang = await cli.prompt('Language: ', {default:'English'});
```
Comment (hliyan) [inline test.js]: What's this for?

File: index.js [hunk 3]
```diff
@@ -30,17 +30,24 @@ const getState = (key) => {
   return state[key]; // WARNING: mutable
 };
 
+const _prompt = async (prompt, customConf, message) => {
```
Comment (hliyan) [inline index.js]: Feels like an extra level of indirection... can't we roll this into `prompt()` itself?

File: index.js (hunk 1 above)
Comment (chamathb) [reply index.js]: Done

File: index.js (hunk 3 above)
Comment (chamathb) [reply index.js]: Removed this and done the required changes to prompt method itself

**Pull request merged**